Lib/test/test_eof.py

```diff
@@ -0,0 +1,77 @@
+"""test script for a few new invalid token catches"""
+
+import sys
+from test import support
+from test.support import os_helper
+from test.support import script_helper
+import unittest
+
+# TODO: RUSTPYTHON
+@unittest.expectedFailure
```

Comment:
@youknowone I don't think that decorator _can_ be applied on a class, but I could be wrong.

Reply:
It works as we expect. Removing it results all test fail.
On the other hand, applying them on function looks more useful to support issue hunters though.

Reply:
Either way, these lines should have been broken out into a separate commit.